about.css

```diff
@@ -0,0 +1,106 @@
+main h1 {
+  background-image: none;
+  -webkit-text-fill-color: rgb(236, 82, 66);
+  font-size: calc(1.425rem + 2.1vw);
+  padding-bottom: 1.5rem;
+  font-weight: 700;
+}
+
+.free-courses h2 {
+  font-family: 'Ubuntu', sans-serif !important;
+  font-size: 25px;
+}
+
+main h2:nth-last-child(2) {
+  color: #3d3d3d;
+  font-weight: 300;
+  line-height: 1.2;
+  letter-spacing: 4px;
+}
+
+p:nth-last-child(3) {
+  background-color: #f7f7f7;
+  font-size: 20px;
+  color: #5e5e5e;
+  flex: 1 1 auto;
+  padding: 1rem 1rem;
+  line-height: 1.4;
+  letter-spacing: 4px;
+}
+
+main h3 a {
+  font-weight: 700;
+  color: #3d3d3d;
+  text-align: center;
+}
+
+@media (max-width: 300px) {
+  main h3 a {
+    font-size: 20px;
+    font-weight: 500;
+  }
+}
+
+.access-winner {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding: 30px;
+}
+
+.access-winner p {
+  background-color: white;
+  font-size: 20px;
+}
+
+.access-winner .winner-card {
+  width: 90%;
+  padding: 30px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  box-shadow: 1px 1px 2px 2px #888;
+}
+
+.free-courses {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 30px;
+}
+
+.free-courses p {
+  background-color: white;
+  font-size: 30px;
+  font-family: 'Lato', serif;
+  margin-bottom: 10px;
+}
+
+.course {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  background-image: url(./images/corse.webp);
+  background-position: center;
+  background-size: cover;
+  background-color: rgba(255, 0, 0, 0.5) !important;
```

Comment:
- [ ] Using `!important` is bad practice and should be avoided because it makes debugging more difficult by breaking the natural cascading in your stylesheets.